lib/aca_entities/families/family_reference.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module AcaEntities
```

Comment:
Reference models are intended to support humans as they interpret data payloads.  Therefore, there's no reason to create a reference class with only one attribute.  

Consider adding of other attributes (e.g. the primary_member's hbx_id) or replace with family_hbx_id in entities

Reply:
Updated to `family_hbx_id`